.cypress/cypress/integration/staff.js

```diff
@@ -43,4 +43,52 @@ describe('Staff user tests', function() {
         cy.get('[name=detail]').should('have.value', 'A Graffiti (offensive) problem has been found by Borsetshire County Council');
         cy.cleanUpXHR();
     });
+
+    it('does not let staff update their name, phone or email address whilst reporting or updating', function() {
+        // Lest CS staff forget to select 'report as another user' and type the reporter's details into their own account.
+        cy.server();
+
+        Cypress.on('uncaught:exception', function(err, runnable) {
```

Comment:
"All visits to /report/new with a lat and long in existing tests were to cobrands with a mocked asset layer" - even ignoring the many other test files that don't mock an asset layer (e.g. category_tests.js or regressions.js), the first test in this file does not mock an asset layer and progresses successfully to test a "Report as" box, which is practically the same thing you wish to be testing. You don't need a lat/lon for what you want to test, you could copy that existing test set up (or even perhaps have a shared setup function), that's why I suggested eg "[use] the same method of logging in".

We do not want a "Cypress.on" global exception - we want the tests to fail if there are any errors. At best as per https://docs.cypress.io/api/events/catalog-of-events#To-catch-a-single-uncaught-exception you would want to use `cy.on`, but this is not necessary because you don't need to visit an Isle of Wight location or anything like that. Replacing all the code from `Cypress.on` to the Flytipping selection line with a copy of `cy.route` to `cy.pickCategory` (with the different user) from the test just above this test, the remainder of your test appears to continue to pass successfully, without the need for special exception handling.

Reply:
Thanks for clarifying. I didn't think the global exception was appropriate – I just wanted to push up some code that didn't have a broken test. I should have made clear I didn't regard this as a proper solution by adding 'WIP'.